advocacy_docs/dev-guides/deploy/windows.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Installing PostgreSQL for development and testing on Microsoft Windows
+navTitle: On Windows
+description: Install PostgreSQL on your local Windows machine for development purposes.
+product: postgresql
+iconName: logos/Windows
+---
+
+## Prerequesites
+
+- Windows 10 build 16299 or later
+- An account with Administrator rights
+
+## Installing
+
+For a development machine, we'll use [WinGet](https://learn.microsoft.com/en-us/windows/package-manager/winget/) to download and install PostgreSQL:
+
+```
+winget install PostgreSQL.PostgreSQL.16
+__OUTPUT__
+Found PostgreSQL 16 [PostgreSQL.PostgreSQL.16] Version 16.4-1
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://get.enterprisedb.com/postgresql/postgresql-16.4-1-windows-x64.exe
+  ██████████████████████████████   356 MB /  356 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+...where `16` is the major version of PostgreSQL we wish to install. This will install PostgreSQL in unattended mode, putting it into the default location (`C:\Program Files\PostgreSQL\`) with the default password (`postgres`) and installing both StackBuilder and PGAdmin.
+
+!!! Tip
+
+You can find a list of available PostgreSQL versions by using WinGet's `search` command:
+
+```
+winget search PostgreSQL.PostgreSQL
+___OUTPUT___
+Name          Id                       Version Source
+------------------------------------------------------
+PostgreSQL 10 PostgreSQL.PostgreSQL.10 10      winget
+PostgreSQL 11 PostgreSQL.PostgreSQL.11 11      winget
+PostgreSQL 12 PostgreSQL.PostgreSQL.12 12.20-1 winget
+PostgreSQL 13 PostgreSQL.PostgreSQL.13 13.16-1 winget
+PostgreSQL 14 PostgreSQL.PostgreSQL.14 14.13-1 winget
+PostgreSQL 15 PostgreSQL.PostgreSQL.15 15.8-1  winget
+PostgreSQL 16 PostgreSQL.PostgreSQL.16 16.4-1  winget
+PostgreSQL 9  PostgreSQL.PostgreSQL.9  9       winget
+```
+
+Installers for release candidate versions of PostgreSQL are also available on EDB's website at https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
```

Comment:
I don't understand why this statement is here. The Tip is about WinGet's `search` command, is this pointing me to another place where I can get the same thing or something? I am confused.

Reply:
WinGet provides quick access to the latest major version installers going back to PG 9 - but EDB builds installers for pre-release versions as well. Right now, a 17 RC 1 installer is available on the downloads page, anticipating the full release of v17 in a week or so.